libs/designsystem/accordion/src/accordion.directive.ts

```diff
@@ -1,8 +1,21 @@
-import { Directive } from '@angular/core';
+import { AfterContentInit, ContentChildren, Directive, Input, QueryList } from '@angular/core';
+import { AccordionItemComponent } from './accordion-item.component'; // Adjust path as needed
 
 @Directive({
   selector: '[kirbyAccordion], kirby-accordion',
   exportAs: 'kirby-accordion',
   standalone: false,
 })
-export class AccordionDirective {}
+export class AccordionDirective implements AfterContentInit {
+  @Input() headingLevel: number;
+  @Input() role: string;
+  @ContentChildren(AccordionItemComponent) listChildren: QueryList<AccordionItemComponent>;
+
+  ngAfterContentInit(): void {
+    if (this.listChildren) {
+      this.listChildren.forEach((child) => {
+        child.headingLevel = this.headingLevel;
+      });
+    }
```

Comment:
Since this component is a parent component containing multiple possible accordion-items that might appear in the template asynchoronously or conditionally, it would be a bit more robust to subscribe to the changes of the QueryList, and set the levels reactively. 

A little info on that can be found [here](https://angular.dev/api/core/QueryList#description).